docs/dockercompose/9-splunk-logging.md

```diff
@@ -0,0 +1,66 @@
+# Logging
+
+The default configuration of docker compose is not sending the logs to Splunk. Container logs can be accessed with command:
+```
+docker logs <container_name/id>
+```
+
+Creating a logs requires updating configuration of several docker compose files. To simplify this process, inside the 
+`docker_compose` package there is a `manage_logs.py` file which will automatically manage logs.
+
+## Prerequisites
+
+Running script requires installation of `ruamel.yaml` package for python. It can be done with command:
+```
+pip3 install ruamel.yaml
+```
+
+The following parameters have to be configured in `.env` file:
+`SPLUNK_HEC_TOKEN`, `SPLUNK_HEC_PROTOCOL`, `SPLUNK_HEC_HOST`, `SPLUNK_HEC_PORT`, `SPLUNK_LOG_INDEX`, `SPLUNK_HEC_INSECURESSL`.
+
+More about `.env` configuration can be found in [.env file configuration](./6-env-file-configuration.md).
```

Comment:
maybe we could add description of these variables in `/6-env-file-configuration.md` as well?

Reply:
good point